capstone-project/src/main/angular-webapp/src/app/app.module.ts

```diff
@@ -0,0 +1,30 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { HttpClientModule } from '@angular/common/http';
+import { GoogleMapsModule } from '@angular/google-maps';
+import { MatSliderModule } from '@angular/material/slider';
+
+
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+import { MapComponent } from './map/map.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
```

Comment:
Remove empty line

Reply:
Same, that the style I saw in all the examples